src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -52,6 +52,18 @@
     short-summary: Manage a collection of steps for building, testing and OS & Framework patching container images using Azure Container Registries.
     """
 
+helps['acr run'] = """
+    type: command
+    short-summary: Queues a quick run providing streamed logs for an Azure Container Registry.
+    examples:
+        - name: Queue a local context, pushed to ACR with streaming logs.
+          text: >
+            az acr run -r myregistry --file bash-echo.yaml .
```

Comment:
update to `-f` since you've fixed that here

Reply:
Fixed! Thanks for pointing this out.